modmesh/system.py

```diff
@@ -83,6 +82,19 @@ def _run_viewer(argv):
     wm.windowTitle = "Modmesh Viewer"
     wm.resize(w=1000, h=600)
     wm.show()
+
+    config = {
+        'appEnvName': 'main',
```

Comment:
We generally do not use `camelCase`.  In modmesh, only GUI related code uses `camelCase` because that is Qt convention.  Everywhere else we should use `snake_case`.

Reply:
And please keep the name `master`.  Do not use `main`.  I do not like the pronunciation of `main`, which usually get confused with `M`.